modules/openapi-generator/src/main/resources/typescript-nestjs/README.mustache

```diff
@@ -1,4 +1,4 @@
-## {{npmName}}@{{npmVersion}}
+modules/openapi-generator/src/main/resources/typescript-nestjs/README.mustache {{npmName}}@{{npmVersion}}
```

Comment:
@wing328 was this change intentional? the `##` would make this a sub-title

Reply:
@macjohnny thanks for catching it. Looks like a mistake in my side. Will file a PR to fix it.

Reply:
Filed https://github.com/OpenAPITools/openapi-generator/issues